os/kernel/binary_manager/binary_manager_load.c

```diff
@@ -622,11 +624,17 @@ int binary_manager_read_header(char *path, binary_header_t *header_data, bool cr
 	}
 
 	if (crc_check) {
+		crc_bufsize = header_data->bin_ramsize;
```

Comment:
`header_data->bin_ramsize` is unsigned type, but `crc_bufsize` is signed.
I think it is better to declare `crc_bufsize` to unsigned.

Reply:
modified